src/styles/header.scss

```diff
@@ -0,0 +1,59 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 100%;
+
+}
+
+.nav {
+  margin-left: 50px;
+
+  &__list {
+    display: flex;
+    align-items: center;
+    height: 60px;
+
+    list-style: none;
+    text-transform: uppercase;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    position: relative;
+    padding: 23px 0;
+
+    text-decoration: none;
+    cursor: pointer;
+    color: $titleTextColor;
+    transition-duration: $animationTransitionDuration;
+
+    &:hover {
+      color: $secondaryColor;
+    }
+  }
+}
+
+.is-active {
+  color: $secondaryColor;
+
+  &::after {
```

Comment:
You are treating `is-active` as a block

If it is a block - it should be in a separate file
But why you have made it a block? Please explain, if you will keep it a block.